mlrun/api/db/sqldb/db.py

```diff
@@ -3820,3 +3821,22 @@ def set_run_notifications(
                 project=project,
             )
         self._commit(session, [run], ignore=True)
+
+    def drop_table(self, session: Session, table_name: str):
+        """Drop a table from the database
+
+        :param session: SQLAlchemy session
+        :param table_name: Table name
+        """
+        metadata = sqlalchemy.MetaData(bind=session.bind)
+        keys = metadata.tables.keys()
+        print(keys)
+
+        metadata.reflect()
+        keys = metadata.tables.keys()
+        print(keys)
+
+        # reflect the table using the metadata object
+        table = metadata.tables.get(table_name)
+        if table is not None:
```

Comment:
we might want an exception here if the table doesn't exist. Maybe a flag of `raise_on_not_exists=True` and set it to `False` if you are ok with the action being "idempotent"